recipes/eagle2/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: eagle2
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('eagle2', max_pin='x.x.x') }}
+
+source:
+  url: https://github.com/poruloh/Eagle/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 85804bfe972186ccb66e602856d6e04044302e5c0dbf9309849b0cde981e3432
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - htslib >=1.9
+    - bzip2
+    - zlib
+    - boost-cpp >=1.58
+    - openblas >=0.2.19
+  run:
+    - {{ pin_compatible('htslib') }}
+    - {{ pin_compatible('bzip2') }}
+    - {{ pin_compatible('boost-cpp') }}
+    - {{ pin_compatible('zlib') }}
+    - {{ pin_compatible('openblas') }}
+
+test:
+  commands:
+    - eagle -h
+    - ls
+    - eagle --vcfRef=Eagle-{{ version }}/example/ref.bcf --vcfTarget=Eagle-{{ version }}/example/target.vcf.gz --geneticMapFile=Eagle-{{ version }}/tables/genetic_map_hg19_example.txt.gz --outPrefix=target.phased
+    - eagle --bfile=Eagle-{{ version }}/example/EUR_test --geneticMapFile=USE_BIM --chrom=21 --outPrefix=phased --numThreads=4 --v1fast
+about:
+  home: https://github.com/poruloh/Eagle
+  license: GPL-3.0
```

Comment:
Please use SPDX identifiers

Reply:
In the documentation the license is GNU General Public License v3.0 (GPLv3).
Should I use GPL-3.0-only or GPL-3.0-or-later ?